### PR TITLE
November update for OZW

### LINKED ISSUE #1582: OpenZWave 0.6.0 not starting due to missing vnc_lite.html
## The problem

Just upgraded to 0.6.0. The previous version was working fine. Now getting this output:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] auth.sh: executing... 
[23:19:55] INFO: Auth database exists
[cont-init.d] auth.sh: exited 0.
[cont-init.d] mqtt.sh: executing... 
[23:19:56] INFO: Connected to internal MQTT service
[cont-init.d] mqtt.sh: exited 0.
[cont-init.d] network_key.sh: executing... 
[cont-init.d] network_key.sh: exited 0.
[cont-init.d] novnc.sh: executing... 
sed: /usr/share/novnc/vnc_lite.html: No such file or directory
[cont-init.d] novnc.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] mqtt.sh: executing... 
[23:19:57] INFO: Ensure upstream MQTT server has the correct OZW status
[cont-finish.d] mqtt.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

## Environment

- Add-on with the issue: OpenZWave
- Add-on release with the issue: 0.6.0
- Last working add-on release (if known): 0.5.4
- Operating environment (OS/Supervised): HassOS 4.13

## Problem-relevant configuration

```yaml

```

## Traceback/Error logs

```txt

```

### PR DESCRIPTION
**Pull request by pvizeli**
- OZW xml updates
- OZW-QT bugfixes
- Fix VNC

fixes #1582 
Superseeds and closes #1674

### CODE REVIEW

**Comment by ludeeus** [comment]
Does this PR fix/close any of the open issues?
<https://github.com/home-assistant/addons/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22add-on%3A+zwave%22+>

**Comment by MartinHjelmare** [comment]
Added the known fix.

**Comment by MartinHjelmare** [comment]
I'll test this PR after lunch.

File: zwave/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.7.0
+
+- Fix vnc / ozwadmin
+- Update Openzwave to 1c7b620
+- Update qt-openzwave to 7ebd43e2
+
 ## 0.5.2
```
Comment (MartinHjelmare) [inline zwave/CHANGELOG.md:9]: Should we keep this in the changelog?

**Review by MartinHjelmare — COMMENTED** [review]
I've tested this on my Odroid N2. It works well with my devices.

I've tested:
sensor
light: on/off, dimming, color
switch
lock: open/close
ozwadmin: reading and writing values

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**